### DISCUSSION TITLE
ddns-go导致nas的ipv6变动频繁

### DISCUSSION THREAD

**Discussion by acharnold** [jeessy2/ddns-go#983] [Q&A]
大家遇到过如下情况没有:
威联通nas中docker 运行ddns-go后，ipv6 改变非常频繁，基本5分钟一次，导致DNS解析不正确(一天更新了2w多次，初步怀疑运营商缓存更新不及时），停止ddns-go后ipv6地址变动恢复正常，半天或一天一次

**Comment by Separatee** [comment]
去路由器或者Nas后台修改网络设置
Mac使用设备Mac,ip改成静态。
